zerver/webhooks/teamcity/tests.py

```diff
@@ -14,6 +14,10 @@ def test_teamcity_success(self) -> None:
         expected_message = u"Project :: Compile build 5535 - CL 123456 was successful! :thumbs_up:\nDetails: [changes](http://teamcity/viewLog.html?buildTypeId=Project_Compile&buildId=19952&tab=buildChangesDiv), [build log](http://teamcity/viewLog.html?buildTypeId=Project_Compile&buildId=19952)"
         self.send_and_test_stream_message('success', self.SUBJECT, expected_message)
 
+    def test_teamcity_success_branch(self) -> None:
+        expected_message = u"Project :: Compile (MyBranch) build 5535 - CL 123456 was successful! :thumbs_up:\nDetails: [changes](http://teamcity/viewLog.html?buildTypeId=Project_Compile&buildId=19952&tab=buildChangesDiv), [build log](http://teamcity/viewLog.html?buildTypeId=Project_Compile&buildId=19952)"
+        self.send_and_test_stream_message('success_branch', self.SUBJECT, expected_message)
```

Comment:
One more thing, I agree with Tim that we should include the branch name in the topic, not the body of the message. :) It is a convention we follow across all of our integrations that need to include branch names in their notifications! :)

Reply:
Done !